docs/AboutUs.md

```diff
@@ -9,15 +9,14 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ## Project team
 
-### John Doe
+### Kuan Jian Rong
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/jr.png" width="200px">
```

Comment:
What a handsome young man.

Reply:
Perhaps the filename of the profile photo should be kuanjr.png as mentioned in the textbook to name it to github_username_in_lower_case.png